docs/navigation-parameters-reference.md

```diff
@@ -96,6 +96,12 @@ The navigation parameters allow you to configure **global** navigation settings
 - **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.
 - **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.
 
+### tooltipText
+- **type**: string
+- **description**: Add tooltip to link/button of navigation.
```

Comment:
This looks much better with documentation now 👍 . 

I would propose a slightly different sentence as this looks quite shallow, although I'm still struggling to understand the context overall. 

Feel free to reply to my initial comment above, or let's discuss this at a daily. Could be something I am getting wrong.

Reply:
I propose  here `**description**: Add tooltip to navigation element.  If param is set tooltip will show if not tooltip will stay hide. `